de.dlr.sc.virsat.excel.ui/src/de/dlr/sc/virsat/excel/ui/wizards/ExportPage.java

```diff
@@ -188,18 +181,18 @@ private void createTemplateUI() {
 		composite1.setLayout(new GridLayout(ROWS, false));
 		GridData data = new GridData(GridData.HORIZONTAL_ALIGN_FILL | GridData.GRAB_HORIZONTAL);
 		composite1.setLayoutData(data);
-		
+
 		label2 = new Label(composite1, SWT.SHADOW_IN);
 		label2.setText("Select the template ");
 		new Label(composite1, SWT.NONE);
 		new Label(composite1, SWT.NONE);
-		
+
 		label1 = new Label(composite1, SWT.NONE);
-		label1.setText("Template");		
-		
-        // destination name entry field
+		label1.setText("Template");
+
+		// destination name entry field
 		templateField = new Combo(composite1, SWT.SINGLE | SWT.BORDER);
-        data = new GridData(GridData.HORIZONTAL_ALIGN_FILL | GridData.GRAB_HORIZONTAL);
+		data = new GridData(GridData.HORIZONTAL_ALIGN_FILL | GridData.GRAB_HORIZONTAL);

```

Comment:
Tabs/spaces inconsistency. I think we decided on using spaces a while time ago, so let's at least not change it back to tabs.

Reply:
done